csep/utils/plots.py

```diff
@@ -988,40 +989,59 @@ def plot_poisson_consistency_test(eval_results, normalize=False, one_sided_lower
     Note: All of the evaluations should be from the same type of evaluation, otherwise the results will not be
           comparable on the same figure.
 
-    Description of plotting arguments:
-        xlabel (str): the label for the x-axis of the plot
-
-        title (str): title of the plot
-
     Args:
         results: list storing test results  csep.core.evaluations.EvaluationResult (see note above)
         plot_args: optional argument containing a dictionary of plotting arguments
         one_sided_lower (bool): select this if the plot should be for a one sided test
-        normalize (bool): select this if the forecast likelihood should be normalized by the observed likelihood. useful for plotting simulation based simulation tests.
+        normalize (bool): select this if the forecast likelihood should be normalized by the observed likelihood. useful
+                          for plotting simulation based simulation tests.
+
+    Description of optional plotting arguments:
```

Comment:
I do not know to properly write the function description of plot_args, so it automatically goes to the documentation. Any suggestion with this would be cool, for future commits

Reply:
i think the best option is to create a documentation page on the plotting. this requires getting sphinx set up and learning reST, but it would be super helpful to learn how this works. lets chat later this week and i can help you get set up using sphinx and writing documentation

for docstrings you can do it how you are showing, any text in side the """ """ will be rendered using docstrings.